theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts

```diff
@@ -60,6 +60,12 @@ export namespace TraceViewerToolbarCommands {
         label: 'Server health and latency are good. No known issues',
         iconClass: 'fa fa-check-circle-o fa-lg',
     };
+
+    export const TRACE_OVERVIEW: Command = {
```

Comment:
> You should have 2 commands with 2 different icons and label so that it's clear what is done.
> 
> The hide button should show an overlay diagonal strike-through line (from top-left to bottom right) or a overlay x.

Thinking about it a bit more. We don't need to have a separate button for closing the view from the toolbar. The view can already be closed by the x-button in the view title bar. So, in the toolbar you only need to have the show button with the corresponding icon, label and tooltip.